corehq/apps/style/static/style/less/bootstrap3/report.less

```diff
@@ -34,5 +34,5 @@
     height: 100%;
     background-color: @gray;
     // Fade for backdrop
-    .opacity(80);
+    .opacity(0.8);
```

Comment:
Value of opacity is not divided by 100 in .opacity mixin in b3: http://getbootstrap.com/css/#less-mixins-opacity